src/frontend/screens/MapScreen/GPSPermissions/GPSPermissionsModal.tsx

```diff
@@ -38,13 +88,7 @@ export const GPSPermissionsModal = React.memo(() => {
       enableDynamicSizing
       onDismiss={onBottomSheetDismiss}
       handleComponent={() => null}>
-      <BottomSheetView>
-        {isGranted ? (
-          <GPSPermissionsEnabled />
-        ) : (
-          <GPSPermissionsDisabled setIsGranted={setIsGranted} />
-        )}
-      </BottomSheetView>
+      <BottomSheetView>{renderContent()}</BottomSheetView>
```

Comment:
nit: `renderContent()` being a function instead of a variable has a small overhead. probably insignificant in this case but can be somewhat prone to issues if it was more complex